content/programming_model_overview.tex

```diff
@@ -87,7 +87,7 @@
       destination \ac{PE}. 
   \item \OPR{Quiet}: The \ac{PE} calling quiet ensures completion of remote access
       operations and stores to symmetric data objects. 
-  \item \OPR{Barrier}: All or some \acp{PE} collectively synchronize and ensure
+  \item \OPR{Barrier}: All, or some \acp{PE}, collectively synchronize and ensure
```

Comment:
This seems like a misapplication of _Comma Use No. 3_ in [Purdue's Extended Rules for Using Commas](https://owl.english.purdue.edu/owl/resource/607/02/).